pkgs/development/python-modules/tables/default.nix

```diff
@@ -32,20 +31,6 @@ buildPythonPackage rec {
     hash = "sha256-1HAmPC5QxLfIY1oNmawf8vnnBMJNceX6M8RSnn0K2cM=";
   };
 
-  patches = [
-    (fetchpatch {
-      name = "numpy-1.25-compatibility.patch";
```

Comment:
What's the story with the patches? They're not mentioned in the PR.

Reply:
The fixes are already in 3.9.2, so the patches don't apply.